examples/android/routeguide/app/build.gradle

```diff
@@ -52,8 +52,8 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:27.0.2'
 
     // You need to build grpc-java to obtain these libraries below.
-    implementation 'io.grpc:grpc-okhttp:1.51.1-SNAPSHOT' // CURRENT_GRPC_VERSION
-    implementation 'io.grpc:grpc-protobuf-lite:1.51.1-SNAPSHOT' // CURRENT_GRPC_VERSION
-    implementation 'io.grpc:grpc-stub:1.51.1-SNAPSHOT' // CURRENT_GRPC_VERSION
+    implementation 'io.grpc:grpc-okhttp:1.51.1' // CURRENT_GRPC_VERSION
```

Comment:
this looks incorrect.
@ejona86

Reply:
Yes, I seem to have only included the 4 golden files in the commit to go to 1.51.2-SNAPSHOT .